pinot-core/src/test/java/org/apache/pinot/core/data/table/TableResizerTest.java

```diff
@@ -65,6 +76,33 @@ public void setUp() {
         new Key(new Object[]{"c", 50, 4.0}),
         new Key(new Object[]{"c", 300, 5.0})
     );
+
+    // GroupKeys: d1: a0 ~ a14, d2: 10.0 ~ 24.0, d3: 1.0 ~ 15.0
+    _groupKeys = new LinkedList<>();
```

Comment:
Let's make `_groupKeys` and `_groupByResultHolders` consistent with `_records` and `_keys` (5 groups, values in holder the same as in records)

Reply:
Changed _groupKeys and _groupResultholders to align with the old tests